data/json/items/mods.json

```diff
@@ -87,7 +87,7 @@
         "name" : "homemade suppressor",
         "dispersion_modifier" : 15,
         "damage_modifier" : -5,
-        "description" : "Using a suppressor is almost an imperative in zombie-infested regions.  Gunfire is very noisy, and will attract predators.  Its only drawback is a reduced muzzle velocity, resulting in less accuracy and damage.",
+        "description" : "A suppressor reduces the amount of noise and muzzle flash generated by the report of a firearm. Gunfire is extremely loud and can damage your hearing without protection, a suppressor will bring the loudness of a report down to generally safe levels. A suppressor will slightly reduce recoil and muzzle velocity.",
```

Comment:
Small thing, but you should be using two spaces after each period on both this and the normal suppression description.